chain/chain/src/chain.rs

```diff
@@ -208,6 +255,7 @@ pub struct Chain {
     store: ChainStore,
     pub runtime_adapter: Arc<dyn RuntimeAdapter>,
     orphans: OrphanBlockPool,
+    orphans_requested_missing_chunks: HashSet<CryptoHash>,
```

Comment:
I wonder whether it make sense to make this part of `OrphanBlockPool`

Reply:
done